microservices/twitch/src/twitch.go

```diff
@@ -205,3 +217,14 @@ func GetInstanceId() string {
 	log.Printf("The instance id is %s\n", out)
 	return string(out)
 }
+*/
+func GetInstanceId() string {
```

Comment:
The original `GetInstanceId` function should be left in for the purpose of this example